source/install.txt

```diff
@@ -10,168 +10,406 @@ Install Compass
+       name: macOS
+       content: |
+
+     - id: windows
+       name: Windows
+       content: |
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
+
+
+     - id: rhel
+       name: RHEL
+       content: |
+
+
+Prerequisites
 -------------
 
-|compass| works with MongoDB Server version 3.0 and higher, both
-Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+Software Requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. tabs::
+
+   hidden: true
+
+   tabs:
+     - id: osx
+       name: macOS
+       content: |
+
+         |compass-short| requires the 64-bit version of macOS 10.10 or
+         later and MongoDB 3.0 or later.
+
+     - id: windows
+       name: Windows
+       content: |
+
+         |compass-short| requires the 64-bit version of Microsoft
+         Windows 7 or later and MongoDB 3.0 or later.
+
+         .. note::
+
+            The Windows installation of |compass| requires **Microsoft
+            .NET Framework version 4.5 or later.** The |compass-short|
+            installer prompts you to install the minimum required
+            version of the .NET framework if it is not already installed
+            on your system.
+
+     - id: ubuntu
+       name: Ubuntu
+       content: |
+
+         |compass-short| requires the 64-bit version of Ubuntu 14.04 or
+         later and MongoDB 3.0 or later.
 
-Software requirements to run |compass-short| are listed below:
+     - id: rhel
+       name: RHEL
+       content: |
 
-.. include:: /includes/fact-compass-platforms.rst
+         |compass-short| requires the 64-bit version of RHEL 7+ or later
+         and MongoDB 3.0 or later.
 
-Installation
-------------
+Network Requirements
+~~~~~~~~~~~~~~~~~~~~
 
-.. note::
+Certain features of |compass|, such as third-party mapping services and
+Intercom, require inbound connections from external sources to
+function. In some instances, users have reported issues with their
+system firewall blocking these network requests and thereby limiting
+|compass| functionalities. To fix this issue, configure your system
+firewall to allow incoming connections for |compass|.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+Ensure ports ``80`` and ``443`` are open between |compass| and the
+host for the MongoDB Instance to which Compass connects.
+
+.. include:: /includes/fact-isolated-edition-firewall.rst
+
+Download Compass
+----------------
+
+You can use your web browser or the appropriate command line option to
+download |compass-short|.
+
+.. admonition:: |compass-short| bundled with MongoDB 3.6 and later
+   :class: important
+
+   |compass-short| is bundled with the MongoDB Server. If you choose 
```

Comment:
MongoDB Server *download* ?

Reply:
Agree with @jeff-allen-mongo. Or "is included as part of your MongoDB Server download", or something like that.

Reply:
I think it's a bit awkward to jump from a sentence ("You can use your web browser or the appropriate command line option to download Compass.") to an admonition to tabs. The admonition I think just devours the lead-in sentence.

I think the admonition should be the first thing under "Download Compass", and then the sentence "You can use your web browser or the appropriate command line option to download Compass.", and then present the tabs directly below that. (So basically, switching the admonition and the first sentence of this section, _given_ that we've removed the OS-specific sentences that I reference in the comment directly below this one). @atsansone @jeff-allen-mongo